configs/defs/rocky.textproto

```diff
@@ -1384,3 +1384,4017 @@ benchmark_configs: {
+      "                        expected_regex: \"\\\\s*-a\\\\s*always,exit\\\\s*-F\\\\s*arch=b[2346]{2}\\\\s*-S\\\\s*([a-zA-Z_]+,)*query_module(,[a-zA-Z_]+)*\\\\s*-F\\\\s*auid>=1000\\\\s*-F\\\\s*auid!=(unset|-1|4294967295)\\\\s*(-F\\\\s*key=|-k)\\\\s*[!-~]*\\\\s*\""
+      "                    }"
+      "                }"
+      "            }"
+      "        }"
+      "    }"
+      "}"
+      
+  }
+}
+benchmark_configs: {
+  id: "ensure-audit-config-immutable"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
+    title: "Ensure the audit configuration is immutable"
+    description: 
+      "Set system audit so that audit rules cannot be modified with auditctl . Setting the flag \"-e "
+      "2\" forces audit to be put in immutable mode. Audit changes can only be made on system "
+      "reboot.\n"
+      "Note: This setting will require the system to be rebooted to update the active auditd "
+      "configuration settings."
+    rationale:
+      "In immutable mode, unauthorized users cannot execute changes to the audit system to "
+      "potentially hide malicious activity and then put the audit rules back. Users would most "
+      "likely notice a system reboot and that could alert administrators of an attempt to make "
+      "unauthorized audit changes."
+    remediation:
+      "Edit or create the file /etc/audit/rules.d/99-finalize.rules and add the line -e 2 at "
+      "the end of the file:\n"
+      "Example:\n"
+      "```\n"
+      "# printf -- \"-e 2\" >> /etc/audit/rules.d/99-finalize.rules\n"
+      "```\n"
+      "Load audit rules\n"
+      "Merge and load the rules into active configuration:\n"
+      "```\n"
+      "# augenrules --load\n"
+      "```\n"
+      "Check if reboot is required.\n"
+      "```\n"
+      "# if [[ $(auditctl -s | grep \"enabled\") =~ \"2\" ]]; then printf \"Reboot "
+      "required to load rules\\\\n\"; fi\n"
+      "```\n"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+    "generic:{"
+      "    check_alternatives:{"
+      "        file_checks:{"
+      "            files_to_check:{"
+      "                files_in_dir:{"
+      "                    dir_path:\"/etc/audit/rules.d\""
+      "                    recursive: true"
+      "                    files_only: true"
+      "                    filename_regex: \".*\\\\.rules\""
+      "                }"
+      "            }"
+      "            content_entry:{"
+      "                match_type: ALL_MATCH_ANY_ORDER"
+      "                match_criteria:{"
+      "                    filter_regex: \"\\\\s*-e\\\\s+2\""
+      "                    expected_regex: \"^\\\\s*-e\\\\s+2\\\\s*$\""
+      "                }"
+      "            }"
+      "        }"
+      "    }"
+      "}"      
+  }
+}
+benchmark_configs: {
+  id: "ssh-x11-forwarding-disabled"
```

Comment:
We already have a check for this in defs/generic_linux.textproto. If this is exactly the same check then we should just reference the one from generic_linux.